app/packs/stylesheets/dropzone.scss

```diff
@@ -142,16 +144,10 @@
   pointer-events: none;
   opacity: 0;
   z-index: 500;
-  position: absolute;
   display: block;
-  top: 50%;
-  left: 50%;
-  margin-left: -27px;
```

Comment:
This is about errors, can it be reverted?

Reply:
it can be, but ... should it be?    There was code in main that dealt with the error case and stomps on dropzone default behavior ... so ... 🤷🏻

Reply:
that is ... it's not like this will harm anything, and the current behavior is not good in any case, and I made ticket #1577 ...

Reply:
I'm fine if this is in the PR for 1577, but it's confusing when you look at history and see that this is about fixing the checkmark, but it's adjusting styles that are unrelated.  Let's keep each change in its own PR.

Reply:
actually, if you look at the whole file, these particular changes pertain to both success-mark and error-mark.

Reply:
Got it, github had folded up the end of the last block.